config/jest/server/start.mjs

```diff
@@ -10,9 +10,12 @@ const { PORT = configPaths.ports.test } = process.env
 export default async function serverStart () {
   const [server] = await getServers()
```

Comment:
Is there ever a chance (race condition with tests re-running quickly in watch mode or something else) we could have more than one server up? I'm guessing thanks to the `await server.destroy()` that's been introduced there's only ever going to be one server in that list, just wanted to double-check 😄

Reply:
Multiple servers are supported but we only use one

I've had a look and calling [`server.destroy()`](https://github.com/smooth-code/jest-puppeteer/blob/master/packages/spawnd/src/index.js#L27) _does_ actually fully exit the process

So instead of checking if the first `[0]` process in **getServers()** is "exited but not removed", it's probably easier to remove all of them via **serverStop()** instead. I'll add it to the next push

(Just in case multiple servers ever get started)